protected/humhub/modules/content/components/ContentActiveRecord.php

```diff
@@ -211,24 +214,24 @@ public function getOwner()
 
     /**
      * Related Content model
-     * 
-     * @return Content
+     *
+     * @return \yii\db\ActiveQuery|ActiveQueryContent
      */
     public function getContent()
     {
-        return $this->hasOne(Content::className(), ['object_id' => 'id'])->andWhere(['content.object_model' => self::className()]);
+        return $this->hasOne(Content::className(), ['object_id' => 'id'])
+            ->andWhere(['content.object_model' => self::className()]);
     }
 
     /**
      * Returns an ActiveQueryContent to find content.
-     * 
+     *
      * @inheritdoc
-     * 
      * @return ActiveQueryContent
      */
     public static function find()
     {
-        return Yii::createObject(ActiveQueryContent::className(), [get_called_class()]);
+        return new ActiveQueryContent(get_called_class());
```

Comment:
To fix
![image](https://cloud.githubusercontent.com/assets/874234/24185083/5257559a-0ee2-11e7-83f9-83b8c0737c5c.png)